Tests/ValetIntegrationTests/ValetIntegrationTests 2.swift

```diff
@@ -0,0 +1,738 @@
+//
```

Comment:
Did you accidentally duplicate this file?  Not sure how this compiled since there's now 2 `ValetIntegrationTests` classes...

Reply:
Indeed 🤦‍♂️. I’ve since removed the duplicate from the developer branch.